composer.json

```diff
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "yiisoft/yii2": ">=2.0.4",
+        "yiisoft/yii2": "*",
```

Comment:
It need to be a master branch, because it depends on changes made recently.
What whould be correct value then?
During 2.0.5 release it should be set to ">=2.0.5", but what value should be used until then?